src/ripple/rpc/handlers/AccountCurrenciesHandler.cpp

```diff
@@ -52,6 +53,14 @@ Json::Value doAccountCurrencies (RPC::Context& context)
     if (auto jvAccepted = RPC::accountFromString (accountID, strIdent, bStrict))
         return jvAccepted;
 
+    auto const sleAccepted = ledger->read(keylet::account(accountID));
+    if (! sleAccepted)
+    {
+        result[jss::account] = context.app.accountIDCache().toBase58 (accountID);
```

Comment:
fair enough - I'll just switch to `rpcError()` since that's what most methods are using.